app/src/main/java/com/nsc/covidscore/GpsPageFragment.java

```diff
@@ -0,0 +1,91 @@
+package com.nsc.covidscore;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.fragment.app.Fragment;
+
+/**
```

Comment:
I kept it because I thought we might need it if we added functionality for user to later decide to determine location via GPS, but even then we won't need a screen for that. I'll delete